DESCRIPTION

```diff
@@ -1,16 +1,28 @@
 Package: ReactomeGSA
 Type: Package
 Title: Client for the Reactome Analysis Service for comparative multi-omics gene set analysis
+Author: Johannes Griss [aut, cre] (<https://orcid.org/0000-0003-2206-9511>)
+Maintainer: Johannes Griss <johannes.griss@meduniwien.ac.at>
 Version: 1.17.3
 Authors@R: person("Johannes", "Griss", email = "johannes.griss@meduniwien.ac.at", role = c("aut", "cre"), comment = c(ORCID = "0000-0003-2206-9511"))
 Description: The ReactomeGSA packages uses Reactome's online analysis service to perform a multi-omics
     gene set analysis. The main advantage of this package is, that the retrieved results can be 
-    visualized using REACTOME's powerful webapplication.
-    Since Reactome's analysis service also uses R to perfrom the actual gene set
-    analysis you will get similar results when using the same packages (such as limma and edgeR) locally.
+    visualized using REACTOME's powerful web application.
+    Since Reactome's analysis service also uses R to perform the actual gene set
+    analysis, you will get similar results when using the same packages (such as limma and edgeR) locally.
     Therefore, if you only require a gene set analysis, different packages are more suited.
 License: MIT + file LICENSE
 Encoding: UTF-8
+Imports: 
```

Comment:
This leads to duplicate `Imports` statements. Please make sure that the latest version also passes `devtools::check()` and `devtools::test()`